rio_tiler/utils.py

```diff
@@ -657,6 +670,7 @@ def create_cutline(
     src_dst: Union[DatasetReader, DatasetWriter, WarpedVRT],
     geometry: Dict,
     geometry_crs: CRS = None,
+    op: Optional[Callable[[float], Union[int, float]]] = None,
```

Comment:
I feel we could just have `Optional[Callable[[float], Any]]` because the type of `numpy.floor` won't match ☝️